### PR TITLE
OSDOCS-2174: RN for MetalLB

### PR DESCRIPTION
**Pull request by mikemckiernan**
Mention Operator and new CRD.  L2 only for 4.9.  Link to primary information.

Preview link: [MetalLB load balancer](https://deploy-preview-35705--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-networking-metallb).

### CODE REVIEW

**Comment by kmccarron-rh** [comment]
The Special Resource Operator (SRO) is added per https://issues.redhat.com/browse/OSDOCS-2396 as a technology preview.
Also, The Driver Toolkit added to the 4.8 RN post GA will be added to the TP table in 4.9.

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -105,6 +105,17 @@ For more information, refer to xref:../networking/ovn_kubernetes_network_provide
 
 The containerized Data Plane Development Kit (DPDK) is now GA in {product-title} {product-version}. For more information, see xref:../networking/hardware_networks/using-dpdk-and-rdma.adoc#using-dpdk-and-rdma[Using virtual functions (VFs) with DPDK and RDMA modes].
 
+[id="ocp-4-9-networking-metallb"]
+==== MetalLB load balancer
+
+This release introduces the MetalLB Operator. After installing and configuring the MetalLB Operator, you can deploy MetalLB to provide a native load balancer implementation for services on bare-metal clusters. Other on-premise infrastructures that are like bare metal might also benefit.
```
Comment (bmcelvee) [inline release_notes/ocp-4-9-release-notes.adoc:111]: Is there a reason for hyphenating "bare-metal" on the first reference? 

Suggested edit: 
s/might/can

**Comment by bmcelvee** [comment]
A question and a suggestion, otherwise LGTM!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (mikemckiernan) [reply release_notes/ocp-4-9-release-notes.adoc:111]: IIUC, bare metal is used as an adjective to modify the noun, clusters.  I'm attempting to follow the guidance from <https://redhat-documentation.github.io/supplementary-style-guide/#bare-metal-adj>.

**Pull request merged**